app/controllers/runs_controller.rb

```diff
@@ -3,10 +3,10 @@
 require 'speedrundotcom'
 
 class RunsController < ApplicationController
-  before_action :set_run,        only: [:show, :download, :destroy, :compare, :edit, :update]
+  before_action :set_run,        only: %i[show download destroy compare edit update]
   before_action :set_comparison, only: [:compare]
```

Comment:
Does Rubocop not care about `%`-syntax for single-element arrays? & if so do we know why?

Reply:
btw, we can change config file and than we'll write `%i[compare] intead `[:compare]`

Reply:
Unless there are any good arguments against that, let's go ahead and do it for consistency.